<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterDock</class>
 <widget class="QDockWidget" name="ParameterDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>382</width>
         <height>630</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Feature View</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>Show Target Vector Field</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="checkBox_7">
             <property name="text">
              <string>Show Hist Matched Crsp</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="checkBox_5">
             <property name="text">
              <string>Show Scalar Field</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_3">
             <property name="text">
              <string>Show Source Curve</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="checkBox_4">
             <property name="text">
              <string>Show Target Curve</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Show Source Vector Field</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="checkBox_6">
             <property name="text">
              <string>Show Fitted Curve Crsp</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QCheckBox" name="Show_All_Lines_CheckBox">
             <property name="text">
              <string>Show All Constrained Lines</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QCheckBox" name="Show_ProjCrsp_CheckBox">
             <property name="text">
              <string>Show Projection Crsp</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="flatCheckbox">
            <property name="text">
             <string>flat</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="edgeThresholdSlider">
            <property name="value">
             <number>75</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>415</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>98</width>
         <height>28</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
